Representable/representable
# Database diagram and protocols

**Changes**:
- update requirements.txt to include relevant packages for keeping the db diagram up to date
- add current db diagram
- update docs to include protocols for database changes
- update docs to include protocols for code coverage

**To test**:
- pip install -r requirements.txt
- if you have trouble with pygraphviz, try ```brew install graphviz``` and then pip install pygraphviz; if this doesn't work contact me or stackoverflow (this is an intermittent problem with dependencies and installation paths and sometimes incomplete installations, brew fixed it for me, but pip supposedly works for many people).
- run ```python manage.py graph_models -a -o models.png``` and check there are no errors (this should also rewrite models.png, a change which you can discard instead of pushing)
- read over the updated docs and ensure they are free of typos and clear

Closes #402 and closes #403
